src/PostProcesses/postProcess.ts

```diff
@@ -186,6 +188,12 @@ export class PostProcess {
     private _texelSize = Vector2.Zero();
     private _forcedOutputTexture: Nullable<InternalTexture>;
 
+    /**
+    * Prepass configuration in case this post process needs a texture from prepass
+    * @hidden
+    */
+    public prePassEffectConfiguration: PrePassEffectConfiguration;

```

Comment:
If it's hidden, maybe that's because this property is not meant to be used by the end user?

In that case, I think you should prefix it with an underscore.